src/arg_desc.jl

```diff
@@ -94,6 +94,7 @@ const _arg_desc = KW(
 :subplot_index            => "Integer.  Internal (not set by user).  Specifies the index of this subplot in the Plot's `plt.subplot` list.",
 :colorbar_title           => "String.  Title of colorbar.",
 :framestyle               => "Symbol.  Style of the axes frame. Choose from $(_allFramestyles)",
+:camera                   => "NTuple{2, Int(GR)/Real(Other)}. Sets the view angle (azimuthal, elevation) for 3D plots",
```

Comment:
Could we allow `NTuple(2, Real)` in general and round to `Int` for GR?

Reply:
Good idea I'll add that!

Reply:
My initial thought would be to do `map(Int,map(round,sp[:camera]))` that way both a 2Tuple and an Array can work if they are given to `:camera` (`:camera` currently works with both a 2Tuple and Array). Not sure if there is a cleaner way to do this hence I'm posting a comment before I make a commit.

Reply:
Sounds right for me :-)